cmd/mimir/main.go

```diff
@@ -200,7 +205,22 @@ func main() {
 		}
 	}
 
-	t, err := mimir.New(cfg, reg)
+	// Initialize the Alertmanager feature flags. Mimir does not register the flags itself
+	// for the time being. Instead, we initialize the flags to safe defaults, without
+	// for UTF-8 mode.
+	ff, err := featurecontrol.NewFlags(util_log.Logger, featurecontrol.FeatureClassicMode)
```

Comment:
We pass the feature flags from `main.go`. In future, it will be wired up to Mimir's flags.

Reply:
Quickly glancing through this, I'm pretty certain we don't want to conflate the two types of feature flags - Mimir's feature flags (or limits) are of no concern to the Alertmanager.